source/extensions/filters/network/kafka/serialization_composite.h

```diff
@@ -0,0 +1,468 @@
+// XXX this file can be generated, as it's repeating the same code for 0..9 delegates
```

Comment:
You already have all of the jinja setup. Do you want to just do this now? It would also make it simpler to review.